alacritty/src/window.rs

```diff
@@ -261,7 +256,7 @@ impl Window {
             .with_title(title)
             .with_visible(false)
             .with_transparent(true)
-            .with_decorations(decorations)
+            .with_decorations(window_config.decorations != Decorations::None)
```

Comment:
It was suggested by clippy to use `matches!`, but I think PartialEq is better here.

Reply:
Same here, it has nothing to do with the PR.